portal/src/graphql/adminapi/AddEmailScreen.tsx

```diff
@@ -118,12 +118,13 @@ const AddEmailScreen: React.FC = function AddEmailScreen() {
           errorMessage={errorMessage.email}
         />
         <ButtonWithLoading
+          type="submit"
           onClick={onAddClicked}
           disabled={!isFormModified}
           labelId="add"
           loading={creatingIdentity}
         />
-      </section>
+      </form>
```

Comment:
I prefer we pass the same handler to the two components because they are handling the same event.

```typescript
function onFormSubmit(e: React.Syntheticevent<HTMLElement>) {
  e.preventDefault();
  e.stopPropagation();
  // Handle the event.
}

<form onSubmit={onFormSubmit}>
  <ButtonWithLoading onClick={onFormSubmit} />
</form>
```

Reply:
changed to only using onSubmit event